packages/paste-nextjs-template/pages/index.tsx

```diff
@@ -15,51 +13,13 @@ const Home: React.FC = () => {
       </Head>
 
       <Heading as="h1" variant="heading10">
-        Welcome to the{' '}
-        <Anchor href="https://paste.twilio.design" showExternal>
-          Paste Next.js App!
-        </Anchor>
+        Welcome to the the Paste Next.JS App!
       </Heading>
 
       <Paragraph>
         Everything you need to get started using Paste in a Production app. Start by editing{' '}
         <code>pages/index.tsx</code>
       </Paragraph>
-
-      <Separator orientation="horizontal" verticalSpacing="space120" />
-
-      <Heading as="h2" variant="heading20">
-        Useful links
-      </Heading>
-
-      <UnorderedList>
-        <ListItem>
-          <Heading as="h3" variant="heading30">
-            <Anchor href="https://paste.twilio.design" showExternal>
-              Paste Documentation
-            </Anchor>
-          </Heading>
-          <Paragraph>
-            Start here. Find in-depth information about using the Paste Design System to build your Next app.
-          </Paragraph>
-        </ListItem>
-        <ListItem>
-          <Heading as="h3" variant="heading30">
-            <Anchor href="https://nextjs.org/docs" showExternal>
-              NextJS Documentation
-            </Anchor>
-          </Heading>
-          <Paragraph>Find in-depth information about Next.js features and API.</Paragraph>
-        </ListItem>
-        <ListItem>
-          <Heading as="h3" variant="heading30">
-            <Anchor href="https://vercel.com/import?filter=next.js" showExternal>
-              Deploy
-            </Anchor>
-          </Heading>
-          <Paragraph>Instantly deploy your Next.js site to a public URL with Vercel.</Paragraph>
-        </ListItem>
-      </UnorderedList>
```

Comment:
With the fixed esbuild config, is there an opportunity to put this stuff back and it still build and work?

I think there's an opportunity, just like with the token contrast checker, to add a verification step in our build for these templates so we also know they are working.

Reply:
Adding this back in worked, good call.